pkg/metadata/permission.go

```diff
@@ -0,0 +1,9 @@
+package metadata
+
+const (
+	// PermissionModel indicates which role kinds to prefer for resource role
+	// creation. E.g. defining the visibility of a resource as "private" can be
+	// persisted using this metadata key, so that roles can be created according
+	// to the permission model.
+	PermissionModel = "permission.venturemark.co/model"
```

Comment:
@marcusellison I think this should do the trick for persisting the permission model. So you are free to chose how you want to play it. When you want the permission model of a venture to allow members creating timelines, you can give this key any value, e.g. "member" or "allow-timeline-creation". Having this indicator persisted enables you then to create roles accordingly. 

Note that there should be backend validation during role creation, which I do not implement just yet. We can easily add that later once we are up and running.